build.gradle

```diff
@@ -261,11 +261,11 @@ dependencies {
 
   implementation group: 'commons-fileupload', name: 'commons-fileupload', version: '1.5'
 
-  implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.4.14'
-  implementation group: 'ch.qos.logback', name: 'logback-core', version: '1.4.14'
+  implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.10'
+  implementation group: 'ch.qos.logback', name: 'logback-core', version: '1.2.10'
```

Comment:
is there an issue with these versions

Reply:
Yeah the logging seems to be breaking when updating these versions (another reason why I'm turning automerge off for now). Could look into the issue more but to fix the build for now I just reverted them